sphinxcontrib/openapi/renderers/_httpdomain.py

```diff
@@ -315,3 +336,118 @@ def render_response_content(self, media_type, status_code):
             yield f"   Content-Type: {content_type}"
             yield f""
             yield from indented(example.splitlines())
+
+
+def _generate_example_from_schema(schema):
```

Comment:
I think this function might be useful alone and is not generally tied to this project. I'd suggest to put the whole function it some sub-module, let's say `sphinxcontrib/openapi/jsonschemaext.py`. Who knows? Maybe one day this goodie become a separate library or will be incorporated by another.